sbws/lib/v3bwfile.py

```diff
@@ -746,12 +808,30 @@ def from_results(cls, results, scanner_country=None,
         number_consensus_relays = cls.read_number_consensus_relays(
             consensus_path)
         state = State(state_fpath)
+
+        # Create a dictionary with the number of relays excluded by any of the
+        # of the filtering rules that makes relays non-`eligible`.
+        # NOTE: In BW_HEADER_KEYVALUES_RECENT_MEASUREMENTS_EXCLUDED it is
+        # explained what are the KeyValues.
+        # The rules were introduced in #28061 and #27338.
+        # In #28565 we introduce the KeyValues to know why they're excluded.
+        # In #28563 we report these relays, but make Tor ignore them.
+        # This might confirm #28042.
```

Comment:
I don't think you need to repeat this comment?
Repeated comments can get out of sync.
Just say "see from_results() for details"